README.md

```diff
@@ -8,17 +8,34 @@ Uses Bo Hagland's solver https://github.com/dds-bridge/dds -- requires the libdd
 Credit to Alexis Rimbaud of NukkAI for the python dds wrapper.
 
 
-# Build and install libdds library for local testing
+# Build the C++ library for local testing
 
 ```
 make libdds-build
-cd libdds/.build
 ```
 
 The python loader looks for the library from libdds/.build/src. If the file is
 found from build directory then the found library is preferred before a library
 in a system directory.
 
+## Change the build configuring of the C++ library
```

Comment:
Can we incorporate all of this into our Makefile? Executable documentation is my favorite kind.

Reply:
It should be rare that someone would have to change build configuration. But in case there is need to change the configuration then documentation is a helpful tool.

A custom configuration could be required for using -fsanitize compiler options. They allow checking memory access, memory leaks, undefined behavior or thread. We can later automate those use cases. Even then I can see someone rarely needing a custom build configuration (which would then likely need this instructions).

A target to run a configuration tool wouldn't still be missing features. Some people can prefer Qt gui instead of curses one while other people might not have Qt gui installed.
Do you think a make target running the configuration tool would work better than a documentation paragraph?

Reply:
I was confused. This should not be a make target.

As you say, this will be done seldom. The docs could easily confuse a reader, as they did me! I suggest splitting this part out into separate file, perhaps README.configuration.md or README.cmake.md. There you can spell out when someone might want to do it. You could optionally put a link to the new file in the main file.

Reply:
> I was confused. This should not be a make target.
> 
> As you say, this will be done seldom. The docs could easily confuse a reader, as they did me! I suggest splitting this part out into separate file, perhaps README.configuration.md or README.cmake.md. There you can spell out when someone might want to do it. You could optionally put a link to the new file in the main file.

A separate file sounds good. I decided to add `README.libdds.md`.